packages/ui-template/.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
```

Comment:
Remind me again what is the purpose of eslint while we already have `svelte-check`?

Reply:
i was relying on eslint's squiggly lines to highlight issues.

haven't used svelte-check. it looks like a cli to run and check for issues manually? like before a commit?